spec/controllers/admin/domain_blocks_controller_spec.rb

```diff
@@ -33,13 +39,25 @@
   end
 
   describe 'POST #create' do
-    it 'blocks the domain' do
+    it 'blocks the domain when succeeded to save' do
       allow(DomainBlockWorker).to receive(:perform_async).and_return(true)
+
       post :create, params: { domain_block: { domain: 'example.com', severity: 'silence' } }
 
       expect(DomainBlockWorker).to have_received(:perform_async)
+      expect(flash[:notice]).to eq 'Domain block is now being processed'
```

Comment:
I don't think so because:
* The English text is definitive and the test case should be reviewed as well when the text has been changed.
* A test case does not have to be generic.